### PR TITLE
Puzzle et

### PR DESCRIPTION
**Pull request by ellytea**
I refactored Game class and added some function to get it started. With these changes I was able to access the puzzle array from generatePuzzles(). I also did some dom manipulation to display the category. Finally, I created a test file for puzzle and passed the Puzzle method generatePuzzles().
@robbiejaeger @brittanystoroz @thatpamiam

### CODE REVIEW

File: lib/Game.js [hunk 1]
```diff
@@ -2,6 +2,17 @@ class Game {
...
+  }
+
+  nextRound() {
+    this.currentRound++;
+    this.currentPuzzle = this.puzzle.puzzleArray[this.currentRound-1];
+  }
+
+  getCategory() {
+    return this.currentPuzzle.category;
   }
```
Comment (DevelopingDavid) [inline lib/Game.js:16]: loved the way you found a method to get shorten our classes , and add our Round into our game class!! 👍

File: lib/Puzzle.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+      let objs = data.puzzles[puzzle].puzzle_bank;
+      let random = Math.floor(Math.random() * objs.length) + 1;
+      acc.unshift(objs[random]);
+      return acc;
+      }, []);
+  }
+
 
-};
+  };
```
Comment (DevelopingDavid) [inline lib/Puzzle.js:18]: Super stoked that we sticked with making into one method to generate our random puzzles!

File: lib/scripts.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  game = new Game()
+}
+
+// invoke startNewGame on 'Start' button
+startNewGame()
+
+$('.catergory-text').text(game.getCategory());
+
+// $('#startGame').onClick(startNewGame)
+// $('#restartGame').onClick(startNewGame)
```
Comment (DevelopingDavid) [inline lib/scripts.js:13]: Glad there is some pseudo-code,  so we can know what our next issues are for our code-base.

File: style/index.html [hunk 4]
```diff
@@ -117,12 +117,11 @@ <h3 class="player-font player3-total-text">Total Score: 0</h3>
...
-  <script type="text/javascript" src="javascript/Puzzle.js"></script>
-  <script type="text/javascript" src="javascript/Wheel.js"></script>
-  <script type="text/javascript" src="javascript/index.js"></script>
+  <script type="text/javascript" src="../lib/domUpdates.js"></script>
+  <script type="text/javascript" src="../lib/data.js"></script>
+  <script type="text/javascript" src="../lib/Player.js"></script>
+  <script type="text/javascript" src="../lib/Game.js"></script>
+  <script type="text/javascript" src="../lib/Puzzle.js"></script>
+  <script type="text/javascript" src="../lib/Wheel.js"></script>
+  <script type="text/javascript" src="../lib/scripts.js"></script>
```
Comment (DevelopingDavid) [inline style/index.html:126]: Im assuming we're changing the name of the previous 'index.js' to 'scripts.js' to clear some confusion?

File: lib/Game.js [hunk 5]
```diff
@@ -2,6 +2,17 @@ class Game {
   constructor(player1, player2, player3, round = 1) {
     this.players = [];
```
Comment (thatPamIAm) [inline lib/Game.js:3]: With your default paramter of `round = 1`... is there ever an instance of needing to instantiate a new `Game` with something besides that default value? If there isn't (and I can't think of a use case) you would be better off just defaulting your currentRound property to 1.

**Comment by thatPamIAm** [comment]
@ellytea @LittleD1808 You'll want to make sure that you have quite a bit more functionality in place by the end of the weekend - this seems to be a bit behind where you should be at this point in the project.

**Pull request merged**